components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -6813,14 +6796,10 @@ private int getMultiAttributeFilteredUsersCount(Node node, int offset, String do
             AbstractUserStoreManager tempCarbonUM = carbonUM;
             // If domain name are not given, then perform filtering on all available user stores.
             while (tempCarbonUM != null && maxLimit > 0) {
-                // If carbonUM is not an instance of Abstract User Store Manger we can't get the domain name.
-                if (tempCarbonUM instanceof AbstractUserStoreManager) {
```

Comment:
Removed unnecessary if check of `tempCarbonUM instanceof AbstractUserStoreManager` as it is always an instance of `AbstractUserStoreManager`